### ISSUE TITLE
zstask-ctl start: make automatically change_ip

### ISSUE THREAD

**Issue by robertlinux** [zstackorg/zstack#286]
If the saved ip is not available, let zstactk-ctl start change_ip  automatically, this makes the life easier for the one who uses dynamic IP.

**Comment by youyk** [comment]
I don't think this is a good idea to make private cloud management node to use a dynamic IP address. change_ip is a dangrous operation. It should be called with user's manual operation.
